python/lsst/metaserv/schemaToMeta.py

```diff
@@ -47,7 +47,7 @@ class SchemaToMeta(object):
 
     _tableStart = re.compile(r'CREATE TABLE (\w+)*')
     _tableEnd = re.compile(r"\)")
-    _engineLine = re.compile(r'\) (ENGINE|TYPE)=(\w+)*;')
+    _engineLine = re.compile(r'\)[\s]*(ENGINE|TYPE)[\s]*=[\s]*(\w+)*;')
     _columnLine = re.compile(r'[\s]+(\w+) ([\w\(\)]+)')
     _unitLine = re.compile(r'<unit>(.+)</unit>')
     _ucdLine = re.compile(r'<ucd>(.+)</ucd>')
```

Comment:
Further down, beyond the diff in this PR, `_descrMiddle` and `_descrEnd` start with `\s*`, but they're used in a `re.search()` call that can match anywhere in the string.  You need to either change that to `re.match()` so that the search only matches at the beginning of the string or you should remove the `\s*`, since it's essentially useless.